test/fixtures/rules/std_rule_cases/ST10.yml

```diff
@@ -0,0 +1,65 @@
+rule: ST10
```

Comment:
Have you considered the possibility that an "unused" table is actually referenced in a _subquery_? Below is a quick example of what I mean. I don't have a real-world situation in mind -- just threw together this query as an example.

```
SELECT a.col1
FROM a
JOIN b ON a.id = b.a_id
WHERE a.some_column IN (
    SELECT c.some_column
    FROM c
    WHERE c.other_column = b.col
)
```

Reply:
@barrywhart - I've added your query (slightly adjusted) as a test case, and we're all good 👍 . Introspection into subqueries is covered.